infrastructure/demo.tfvars

```diff
@@ -8,3 +8,4 @@ ccd_async_enabled = "false"
 dm_url = "http://dm-store-demo.service.core-compute-demo.internal"
 async_event_operations_enabled = "true"
 save_claim_state_enabled = "true"
+directions_questionnaire_enabled = "true"
```

Comment:
what about mediation toggle? is it already true? i can't see it here..

Reply:
@robertparkinson, i forgot if we have mediation toggle!

Reply:
no mediation toggles in backend - not relevant, they just drive frontend journeys 🤔